README.md

```diff
@@ -386,6 +386,61 @@ Build the binaries as described above, but make sure you unselect `RUN_IN_PLACE`
 Open the generated project file with Visual Studio. Right-click **Package** and choose **Generate**.
 It may take some minutes to generate the installer.
 
+### Compiling on MacOS (Intel Architecture)
```

Comment:
In theory there is nothing preventing this from working on ARM so I'd rather not mention this until we know for sure that it *doesn't* work there.

Reply:
Sure thing, fixed